### PR TITLE
macOS: Support dotnet@6 as installed by Homebrew

### PR DESCRIPTION
**Pull request by Kwarf**
Hi,

On my system I'm running `dotnet@6` installed via Homebrew, which installs in a separate path.
These are the changes I had to make in the app package in order for FamiStudio to locate it and run.

If this is in some way an unsupported setup you can close this PR, but I wanted to push it so anyone else with a similar setup can at least look up how to get it to run. 🙂

Thanks

### CODE REVIEW

**Comment by BleuBleu** [comment]
Ill leave that for now. We are migrating to .NET 7.0 for next version so path may change again. Hit me up if the same issue happens in 4.2.0 (not due for a few weeks/months).

**Comment by BleuBleu** [comment]
Hi. Do you happen to know the install path for .NET 7.0 when installing with homebrew? Id like to include your code in next release, but don't want to change my setup locally.

-Mat

**Comment by Kwarf** [comment]
There's no [official formula](https://github.com/Homebrew/homebrew-core/tree/master/Formula/d) for .NET 7, only 6 and 8, I'm guessing this is because those are LTS releases, and .NET 7 reaches end of support on May 14, 2024.

It might be easiest to just close this and require the runtime to be installed by the official installer, since that appears to handle multiple versions side by side.

**Comment by BleuBleu** [comment]
Got it! Thanks for the information! And yes, Microsoft seems to be in a rush to deprecate versions very quickly, quite hard to follow! 😂

-Mat
